src/service/video/docking.js

```diff
@@ -1504,4 +1509,16 @@ export class VideoDocking {
   hideControlsOnTimeout_(time = CONTROLS_TIMEOUT) {
     this.getHideControlsTimeout_().trigger(time);
   }
+
+  /**
+   * @param {!Element} parent
+   * @private
+   */
+  removePosterForAndroidBug_(parent) {
+    const el = parent.querySelector('.i-amphtml-android-poster-bug');
+    if (!el) {
+      return;
+    }
+    parent.removeChild(el);
```

Comment:
Spoke in person on this one, we should standardize around using the method in the dom library or the native method. The dom library version will compress better today so it's a good reason to use it... in the future it would be great to use builtins and automatically convert them via transforms during the build process.